modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/EnrichmentManagerSpec.scala

```diff
@@ -167,29 +167,32 @@ class EnrichmentManagerSpec extends Specification with EitherMatchers with CatsE
           AcceptInvalid.featureFlags,
           IO.unit,
           SpecHelpers.registryLookup,
-          atomicFieldLimits
+          atomicFieldLimits,
+          emitIncomplete
         )
         .value
-        .map(_ must beLeft.like {
-          case BadRow.SchemaViolations(
-                _,
-                Failure.SchemaViolations(_,
-                                         NonEmptyList(FailureDetails.SchemaViolation.IgluError(schemaKey1, clientError1),
-                                                      List(FailureDetails.SchemaViolation.IgluError(schemaKey2, clientError2))
-                                         )
-                ),
-                _
+        .map {
+          case Ior.Left(
+                BadRow.SchemaViolations(
+                  _,
+                  Failure.SchemaViolations(_,
+                                           NonEmptyList(FailureDetails.SchemaViolation.IgluError(schemaKey1, clientError1),
+                                                        List(FailureDetails.SchemaViolation.IgluError(schemaKey2, clientError2))
+                                           )
+                  ),
+                  _
+                )
               ) =>
-            schemaKey1 must beEqualTo(emailSentSchema)
-            clientError1.toString must contain(
+            schemaKey1 must beEqualTo(AtomicFields.atomicSchema)
+            clientError1.toString must contain("tr_tt")
+            clientError1.toString must contain("Cannot be converted to java.math.BigDecimal")
+            schemaKey2 must beEqualTo(emailSentSchema)
+            clientError2.toString must contain(
```

Comment:
Interestingly the order is not the same when combining several `ValidatedNel[A, B]` and several `Ior[List[A], B]`